rosidl_generator_rs/rosidl_generator_rs/__init__.py

```diff
@@ -219,6 +213,7 @@ def primitive_value_to_rs(type_, value):
     if type_.type in [
             'byte',
             'char',
+            'wchar',
```

Comment:
Honestly, not sure whats going on with this function. My local tests worked just fine without this change. I'm planning to go a bit deeper in a following PR